lib/ecto/changeset.ex

```diff
@@ -236,6 +236,34 @@ defmodule Ecto.Changeset do
         |> Ecto.Changeset.validate_required(...)
         |> Ecto.Changeset.validate_length(...)
 
+  Besides the basic types which are mentioned above, such as `:boolean` and `:string`, 
+  parameterized types can alse be used in schemaless changesets. It's just that parameterized
+  types require a three-element tuple to work:
+
+      {:parameterized, typemod, opts}
+
+  The first element `:parameterized` indicates you want to use a parameterized type.
+  The second element indicates which module to use for handling data. The third element 
+  is the options for this type. 
+
+  Generally, parameterized types will implement `Ecto.ParameterizedType` behaviour. Because
+  of that, in actual usage, we call `Ecto.ParameterizedType.init/2` directly, which will 
+  generate the three-element tuple we need.
+
+  Take `Ecto.Enum`, which implements `Ecto.ParameterizedType`, as an example:
```

Comment:
```suggestion
  parameterized types can also be used in schemaless changesets. They implement
  the `Ecto.ParameterizedType` behaviour and therefore provide an `init/2` function.

  For example, to use `Ecto.Enum` in a schemaless changeset:
```

Reply:
> They implement the `Ecto.ParameterizedType` behaviour and therefore provide an `init/2` function.

The parameterized types that implement the `Ecto.ParameterizedType` behaviour, provides `init/1` instead of `init/2`. 

I changed suggested changes a little bit.